### PR TITLE
Re-architecture project

### LINKED ISSUE #138: Restructure commands handling
The index.ts is loaded with a lot of files inside, it gets hard to read and figure out what is the structure of the command line.

- [x] rearchitecture the index.ts
- [x] split down into commands folder the different groups
- [x] isolate the logic from the commands interface

### PR DESCRIPTION
**Pull request by Mikescops**
Simplify the project structure and make it more coherent.

Fix #138

### CODE REVIEW

File: src/command-handlers/logout.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Database } from 'better-sqlite3';
+import winston from 'winston';
+import { deactivateDevices } from '../endpoints';
+import { connectAndPrepare, connect, reset } from '../modules/database';
+import { Secrets, DeviceConfiguration } from '../types';
+import { askConfirmReset } from '../utils';
+
+export const runLogout = async () => {
+    const resetConfirmation = await askConfirmReset();
+    if (resetConfirmation) {
```
Comment (showi) [inline src/command-handlers/logout.ts:10]: Personaly i prefer `if(!resetConfirmation) return`

Comment (Mikescops) [reply src/command-handlers/logout.ts:10]: absolutely!

File: src/command-handlers/configure.ts [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
-
-export const configureSaveMasterPassword = (params: ConfigureSaveMasterPassword) => {
-    const { db, secrets } = params;
-    let shouldNotSaveMasterPassword = params.shouldNotSaveMasterPassword;
+import { encryptAES } from '../modules/crypto/encrypt';
+import { deleteLocalKey, setLocalKey, warnUnreachableKeychainDisabled } from '../modules/crypto/keychainManager';
+import { connectAndPrepare } from '../modules/database';
+import { parseBooleanString } from '../utils';
+
+export const configureSaveMasterPassword = async (boolean: string) => {
```
Comment (showi) [inline src/command-handlers/configure.ts:7]: `boolean: string` that's funny :]

Comment (Mikescops) [reply src/command-handlers/configure.ts:7]: damn command line ahah

File: src/modules/auth/registerDevice.ts [hunk 3]
```diff
@@ -6,11 +6,11 @@ import {
...
-} from '../endpoints';
-import { askOtp, askToken, askVerificationMethod } from '../utils';
-import { getAuthenticationMethodsForDevice } from '../endpoints/getAuthenticationMethodsForDevice';
-import { requestEmailTokenVerification } from '../endpoints/requestEmailTokenVerification';
-import { SupportedAuthenticationMethod } from '../types';
+} from '../../endpoints';
+import { askOtp, askToken, askVerificationMethod } from '../../utils';
+import { getAuthenticationMethodsForDevice } from '../../endpoints/getAuthenticationMethodsForDevice';
+import { requestEmailTokenVerification } from '../../endpoints/requestEmailTokenVerification';
+import { SupportedAuthenticationMethod } from '../../types';
```
Comment (showi) [inline src/modules/auth/registerDevice.ts:13]: you can add `import type` think about it because you have done it in other files.

**Pull request merged**